python/python/glide/async_commands/redis_modules/json.py

```diff
@@ -139,6 +140,98 @@ async def get(
+) -> TJsonResponse[int]:
+    """
+    Appends the specified `value` to the string stored at the specified `path` within the JSON document stored at `key`.
+
+    See https://redis.io/commands/json.strapped/ for more details.
+
+    Args:
+        client (TRedisClient): The Redis client to execute the command.
+        key (str): The key of the JSON document.
+        value (str): The value to append to the string. Must be wrapped with single quotes.
+        path (Optional[str]): The JSONPath to specify. Default is root `$`.
+
+    Returns:
+        TJsonResponse[int]: For JSONPath (`path` starts with `$`), returns a list of integer replies for every possible path, indicating the length of the resulting string after appending `value`,
+        or None for JSON values matching the path that are not string.
+        For legacy path (`path` doesn't starts with `$`), returns the length of the resulting string after appending `value` to the string at `path`.
+        Note that when sending legacy path syntax, if the JSON value at `path` is not a string or the path doesn't exist, an error is raised.
+        If `key` doesn't exist, an error is raised.
+        For more information about the returned type, see `TJsonResponse`.
+
+    Examples:
+        >>> from glide import json as redisJson
+        >>> import json
+        >>> await redisJson.set(client, "doc", "$", json.dumps({"a":"foo", "nested": {"a": "hello"}, "nested2": {"a": 31}}))
+            'OK'
+        >>> await redisJson.strappend(client, "doc", json.dumps("baz"), "$..a")
+            [6, 8, None]  # The new length of the string values at path '$..a' in the key stored at `doc` after the append operation.
+        >>> await redisJson.strappend(client, "doc", '"foo"', "nested.a")
+            11  # The length of the resulting string after appending " new value" to the string at path 'nested.array' in the key stored at `doc` after the append operation.
+        >>> json.loads(await redisJson.get(client, json.dumps("doc"), "$"))
+            [{"a":"foobaz", "nested": {"a": "hellobazfoo"}, "nested2": {"a": 31}}] # The updated JSON value in the key stored at `doc`.
+    """
+
+    return cast(
+        TJsonResponse[int],
+        await client.custom_command(
+            ["JSON.STRAPPEND", key] + ([path, value] if path else [value])
+        ),
+    )
+
+
+async def strlen(
+    client: TRedisClient,
+    key: str,
+    path: Optional[str] = None,
+) -> TJsonResponse[Optional[int]]:
+    """
+    Returns the length of the JSON string value stored at the specified `path` within the JSON document stored at `key`.
+
+    See https://redis.io/commands/json.strlen/ for more details.
+
+    Args:
+        client (TRedisClient): The Redis client to execute the command.
+        key (str): The key of the JSON document.
+        path (Optional[str]): The JSONPath to specify. Default is root `$`. // TODO check (since when sending strlen "non_exiting" and strlen "non_exiting" . -> behaves the same)
```

Comment:
when running the commands:
```
127.0.0.1:6379> json.strlen non_existing_key
(nil)
127.0.0.1:6379> json.strlen non_existing_key .
(nil)
127.0.0.1:6379> json.strlen non_existing_key $
(error) ERR could not perform this operation on a key that doesn't exist
```

Reply:
so I am not really sure that default it `$`

Reply:
Then remove the comment to what it defaults to, just say in the documentation that if not passed a default value will be used